support-frontend/assets/pages/weekly-subscription-checkout/components/thankYou.jsx

```diff
@@ -114,7 +114,7 @@ function ThankYouContent({
     <div className="thank-you-stage">
       <HeroWrapper appearance="custom" className={styles.hero}>
         <HeroImage />
-        <HeadingBlock overheading="Thank you for supporting our journalism!">
+        <HeadingBlock overheadingClass="--thankyou" overheading="Thank you for supporting our journalism!">
```

Comment:
as stated above this would become `component-heading-block__overheading--thankyou`